server/routerlicious/packages/services-telemetry/src/resources.ts

```diff
@@ -28,6 +28,10 @@ export enum SchemaProperties {
     statusCode = "statusCode",
 }
 
+export enum CommonProperties {
+    restart = "restart",
```

Comment:
Would it make sense to add `restart` as part of `SchemaProperties`? We could also rename `SchemaProperties` to `CommonProperties` in that case

Reply:
Took a stab at reorganizing the enums. Please do let me think if you think we can improve on this.